src/components/BrowseResultsList/BrowseResultsList.test.js

```diff
@@ -103,6 +128,94 @@ describe('BrowseResultsList', () => {
     );
   });
 
+  describe.each([
+    { searchOption: browseCallNumberOptions.CALL_NUMBERS, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.DEWEY, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.LIBRARY_OF_CONGRESS, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.LOCAL, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.NATIONAL_LIBRARY_OF_MEDICINE, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.OTHER, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+    { searchOption: browseCallNumberOptions.SUPERINTENDENT, shared: FACETS.SHARED, heldBy: FACETS.CALL_NUMBERS_HELD_BY },
+  ])('when the search option is $searchOption and the Shared and/or HeldBy facets are selected', ({ searchOption, shared, heldBy }) => {
+    describe('and the user clicks on a record in the list', () => {
+      it('should be navigated to the Search lookup with those filters', async () => {
+        history = createMemoryHistory({
+          initialEntries: [{
+            pathname: BROWSE_INVENTORY_ROUTE,
+            search: `${heldBy}=college&qindex=${searchOption}&query=a&${shared}=true&${shared}=false`,
+          }],
+        });
+
+        renderBrowseResultsList({
+          filters: {
+            qindex: searchOption,
+            query: 'a',
+            [shared]: ['true', 'false'],
+            [heldBy]: ['college'],
+          },
+        });
+
+        await act(async () => fireEvent.click(screen.getByText(defaultProps.browseData[2].fullCallNumber)));
```

Comment:
`fireEvent` is already wrapped in `act` as far as I know, do you wrap it because of asynchronous action?

Reply:
Also worth noting, [userEvent > fireEvent](https://testing-library.com/docs/user-event/intro#differences-from-fireevent) most of the time when you want to do something like "click a button". You don't need to rewrite anything here, but consider it the next time you're writing new tests.

Reply:
@mariia-aloshyna I removed the `act` wrapper.